flutter_modular/lib/src/presenters/navigation/modular_router_delegate.dart

```diff
@@ -69,8 +69,7 @@ class ModularRouterDelegate extends RouterDelegate<ModularRoute>
   }
 
   String resolverPath(String routeName, String path) {
-    final uri = Uri.parse(path);
-    return uri.resolve(routeName).path;
+    return routeName;
```

Comment:
Previous implementation loses the query params information because Uri.path returns only the path without the query params etc. i.e for a url `/signup?email=email.com`, the old implementation returned `/signup` which doesn't have the query params any more. This wasn't caught by the tests because we were passing `/signup?email=email.com` directly into `parser.selectRoute `

I am not sure what this line ` uri.resolve(routeName).path` does tho.

I have updated the tests also.

Reply:
I figured what it does from a failing test. I have updated the implementation to `Uri.toString`